### ISSUE TITLE
Inconsistent use of indentation in docs/installation/local.md

### ISSUE THREAD

**Issue by ashutoshc8101** [fossasia/open-event-server#7619]
**Describe the bug**

Indentation is inconsistent in docs/installation/local.md file

**To Reproduce**
1. Open docs/installation/local.md
2. Scroll to poetry section.

**Expected behavior**
Consistent indentation

**Screenshot**
![issue](https://user-images.githubusercontent.com/24855641/104878808-a4c61e80-5982-11eb-9acd-5bc899f0d4f4.PNG)

**Comment by ria-19** [comment]
Can I work on this issue?

**Comment by ashutoshc8101** [comment]
@ria-19 No use, here maintainers don't review PRs from external developers.
This project is on Github just forsake. This project does not have a single trait of open source projects.
Stop wasting time here.

**Comment by ria-19** [comment]
Okay, I didn't know about this.
Thanks.

**Comment by iamareebjamal** [comment]
Yes, we don't review PRs for issues which are extremely low priority and which are not endorsed or triaged by the maintainers themselves. If you want your PR to be reviewed, please checkout issues labeled codeheat or created by the project maintainers. People have strange notion about open source that they can open any issue they want and send any PR and maintainers have to give time to review and merge them like it's their job

**Comment by ashutoshc8101** [comment]
@iamareebjamal Ok this way, I finally got your attention.
What about this? #7705. This pull request was made after talking about this issue with you.
Note: There were no merge conflicts at the time of PR creation.
![approval](https://user-images.githubusercontent.com/24855641/110324049-789e5400-803b-11eb-8617-dbdd575bbccc.PNG)
You said 'go ahead'  but even after my several requests, you didn't review my PR.
It's not just in open-event-server but in all other fossasia's projects.
Look at https://github.com/fossasia/susi.ai/pull/3630, https://github.com/fossasia/susi.ai/pull/3638, https://github.com/fossasia/susi.ai/pull/3623
There was an issue with pre-deployment with surge which was causing the build to fail. The token was expired.
I reported this to maintainers and even Mario Behling himself.
![Capture](https://user-images.githubusercontent.com/24855641/110325438-5b6a8500-803d-11eb-80f1-510ae5810169.PNG)
They told me to fix that, how the hell can I access your surge account.
And when I requested to review, they got away by saying that your build is failing.
Most of these PRs were solving the issues labeled `bug`.
After two months of contribution to `fossasia`, with 7 PR merged (most of label high priority, bugs). I got this from you guys.
So, finally, I decided to stop contributing to this organization.

I know that it's not your job to review every PR, but it won't kill you to leave a message like `not required, need changes, need improvement` etc. Why leave someone waiting for months. 
Respect other people's work and time too, if not you will deserve only these kind of messages.
Close my other PRs too. 
I don't want to waste time doing that.
I'm done with you guys.

**Comment by ashutoshc8101** [comment]
@iamareebjamal Ok this way, I finally got your attention.
What about this? #7705. This pull request was made after talking about this issue with you.
Note: There were no merge conflicts at the time of PR creation.
![approval](https://user-images.githubusercontent.com/24855641/110324049-789e5400-803b-11eb-8617-dbdd575bbccc.PNG)
You said 'go ahead'  but even after my several requests, you didn't review my PR.
It's not just in open-event-server but in all other fossasia's projects.
Look at https://github.com/fossasia/susi.ai/pull/3630, https://github.com/fossasia/susi.ai/pull/3638, https://github.com/fossasia/susi.ai/pull/3623
There was an issue with pre-deployment with surge which was causing the build to fail. The token was expired.
I reported this to maintainers and even Mario Behling himself.
![Capture](https://user-images.githubusercontent.com/24855641/110325438-5b6a8500-803d-11eb-80f1-510ae5810169.PNG)
They told me to fix that, how the hell can I access your surge account.
And when I requested to review, they got away by saying that your build is failing.
Most of these PRs were solving the issues labeled `bug`.
After two months of contribution to `fossasia`, with 7 PR merged (most of label high priority, bugs). I got this from you guys.
So, finally, I decided to stop contributing to this organization.

I know that it's not your job to review every PR, but it won't kill you to leave a message like `not required, need changes, need improvement` etc. Why leave someone waiting for months. 
Respect other people's work and time too, if not you will deserve only these kind of messages.
Close my other PRs too. 
I don't want to waste time doing that.
I'm done with you guys.

**Comment by MeghalBisht** [comment]
@ashutoshc8101 Why don't you consider rephrasing your comment with less garbage words in it?

**Comment by iamareebjamal** [comment]
I cannot comment about SUSI, as I am not a maintainer there. About your PRs here, #7705 is important and will be merged once we have cleared more important PRs before it. A rule of thumb, consider the PRs with no comment from maintainers are of least priority. As for #7705, I commented `go ahead` means I will eventually merge it. Your PR cannot be merged by just me seeing the code changes. I have to locally pull and run every command to verify if it works or not. And let's say it takes 30 minutes in the process. Right now, I'd rather spend those 30 minutes in fixing a bug which prevents people from editing the profile rather than a PR which changes the system to flask script. 

This is same for PR converting the project to poetry from pip. But I merged it because the script was covered by tests. Your PR changes are not covered by tests, so I have no idea without manual testing if it will break the system. And guess what, even when poetry changes were covered by tests, after I merged it, I spent an entire day fixing the deployment failures and side effect bugs which came with it

For you, it's just one PR, for me, it's a potential system breakage that can happen if I merge it without thoroughly testing, and I simply don't have enough time to do it currently when there are actual usability bugs in the system to be addressed.

Your PR has been open for a month? Guess what, my PRs to prominent projects like django projects have been open for years. I have had to fork projects when they don't merge my PRs. And those are PRs with new features and fixes with 100% code coverage. That's how open source is, buddy. I'll merge your PR once I get time to test it or it has code coverage over the changes you have made. Till then, my focus will be on higher priority bugs
